packages/tizen_bundle/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: tizen_bundle
+description: Tizen bundle APIs.
+homepage: https://github.com/flutter-tizen/plugins
+repository: https://github.com/flutter-tizen/plugins/tree/master/packages/tizen_bundle
+version: 0.0.1
+
+environment:
+  sdk: ">=2.17.0 <3.0.0"
+  flutter: ">=2.5.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  plugin_platform_interface: ^2.0.2
```

Comment:
The dependency `plugin_platform_interface` is unused. Please remove.

Reply:
Done.